root/static/scripts/edit/externalLinks.js

```diff
@@ -209,24 +211,19 @@ export class ExternalLinksEditor
               error = l(`Please select a link type for the URL
                          you’ve entered.`);
             } else if (
-              linkType.deprecated &&
-                (!isPositiveInteger(link.relationship) ||
-                  (oldLink && +link.type !== +oldLink.type))
+              linkType.deprecated && (isNewLink || linkChanged)
```

Comment:
The previous code checked that the type changed, not the URL.

Reply:
Whoops. Created a different constant for this then. What's with the + here though? I just realized those were there. Are they needed and why?